app/assets/stylesheets/style.css

```diff
@@ -143,6 +143,10 @@ a .fa-white,
   min-width: 190px;
 }
 
+#header .navbar #searchform_input:focus {
```

Comment:
I believe this was for https://github.com/publiclab/plots2/issues/9624 and was fixed in https://github.com/publiclab/plots2/pull/9634 but presents a distinct solution from this one: 

https://github.com/publiclab/plots2/pull/9634/files#diff-2b59ed6418df6000b31cea28c6d47e21dc4757cccfffde875b3eb4b1c63909f7R351

@waridrox what do you think? should we merge both? What about getting these two lines closer together in the stylesheet? Thank you all!!!

Reply:
@jywarren, oh no!!! I think in response to some contributor on gitter that the first-timer-only issues are like spoon-feeding I came up with this issue at #9624 stating only what needs to be done 😅 But I think the implementation at #9634 is not correct, nor is it here at #9625. Sorry don't know how this slipped from my sight, but I think we should revert #9634, could that be one of the reasons for intermittent test failures...

Reply:
Ah i see - that is, someone had asked if you could create a slightly more complex first-timers-only issue? I see - well no problem. I think this styling is unlikely to cause any test failures. Do you want to help @jordan-rob figure out the exact place where it should go? Is your GIF above what it /ought/ to look like when it works? 

Thanks, everyone! Appreciate you all!